src/coreclr/src/vm/dllimport.cpp

```diff
@@ -6253,6 +6256,27 @@ namespace
         }
 #endif // FEATURE_CORESYSTEM && !TARGET_UNIX
 
+#if defined(TARGET_LINUX)
+        if (g_coreclr_embedded)
+        {
+            static const LPCWSTR toRedirect[] = {
+                W("System.Native.so"),
+                W("System.IO.Compression.Native.so"),
+                W("System.Net.Security.Native.so"),
+                W("System.Security.Cryptography.Native.OpenSsl.so")
+            };
+
+            int count = sizeof(toRedirect) / sizeof(toRedirect[0]);
```

Comment:
Nit: ~~`sizeof(toRedirect) / sizeof(toRedirect[0])`~~ -> `lengthof(toRedirect)`

Feel free to defer this change until post P8.